src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,43 @@
+.clock {
+  @extend %flex-center;
+  @include circle(640px);
+
+  background-image: url("../images/clock.svg");
+  box-shadow: 4px 4px 16px #ccc;
+
+  transition-property: transform, box-shadow;
+  transition-duration: 0.3s;
+  transition-timing-function: ease-out;
```

Comment:
Just a tip) You can combine all transition properties into one

Reply:
Yes, but than I would need to repeat the duration and the timing function for each property and I am not sure that would be less repetitive, considering that both transitions share these values